### PR TITLE
Ticket 5662: Kepco magnet calibration

### PR DESCRIPTION
**Pull request by DominicOram**
See https://github.com/ISISComputingGroup/IBEX/issues/5662

Moved generic calibration file stuff out of eurotherm and into a separate file

### CODE REVIEW

File: FileListApp/Db/Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ include $(TOP)/configure/CONFIG
 #----------------------------------------------------
 # Create and install (or just install) into <top>/db
 # databases, templates, substitutions like this
-#DB += xxx.db
```
Comment (rerpha) [inline FileListApp/Db/Makefile:13]: should this comment be removed?

File: FileListApp/Db/calibration.db [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    field(LNK2, "$(P)$(CONV_TO_PV):SP:RBV:CONV.NSPE")
+    field(DOL3, "$(P)CAL:SEL")
+    field(LNK3, "$(P)$(CONV_TO_PV):HIGHLIMIT.NSPE")
+    field(DOL4, "$(P)CAL:SEL")
+    field(LNK4, "$(P)$(CONV_TO_PV):LOWLIMIT.NSPE")
+    field(DOL5, "$(P)CAL:SEL")
+    field(LNK5, "$(P)$(CONV_TO_PV):SP:CONV.NSPE")
+    field(DOL6, "1")
+    field(LNK6, "$(P)CAL:SEL:UPDATE:NOW.PROC")
+
```
Comment (rerpha) [inline FileListApp/Db/calibration.db]: some whitespace here

**Review by rerpha — APPROVED** [review]
looks good, works nicely

File: FileListApp/Db/Makefile (hunk 1 above)
Comment (DominicOram) [reply FileListApp/Db/Makefile:13]: I think the comment is just an example of what to do. Now there is a real example the comment seems less necessary

**Pull request merged**